hist/hist/src/TPrincipal.cxx

```diff
@@ -420,17 +420,23 @@ void TPrincipal::AddRow(const Double_t *p)
    }
    else {
 
-      Double_t cor = 1 - 1./Double_t(fNumberOfDataPoints);
+      Double_t invnp = 1. / Double_t(fNumberOfDataPoints);
+      Double_t invnpM1 = 1. /(Double_t(fNumberOfDataPoints - 1));
+      Double_t cor = 1. - invnp;
```

Comment:
```suggestion
      const Double_t invnp = 1. / Double_t(fNumberOfDataPoints);
      const Double_t invnpM1 = 1. /(Double_t(fNumberOfDataPoints - 1));
      const Double_t cor = 1. - invnp;
```
This way it doesn't get overwritten by accident, and it's clear to readers that this is only a temporary helper.